FiveCalls/FiveCalls/ContactLog.swift

```diff
@@ -42,21 +44,23 @@ extension ContactLog : Storable {
             "contactId": contactId,
             "phone": phone,
             "outcome": outcome,
-            "date": ContactLog.dateFormatter.string(from: date)
+            "date": ContactLog.dateFormatter.string(from: date),
+            "reported": reported
         ]
     }
 }
 
 extension ContactLog : Hashable {
     var hashValue: Int {
-        return (issueId + contactId + phone + outcome).hash
+        return (issueId + contactId + phone + outcome + reported.description).hash
     }
     
     static func ==(lhs: ContactLog, rhs: ContactLog) -> Bool {
         return lhs.issueId == rhs.issueId &&
                 lhs.contactId == rhs.contactId &&
                 lhs.phone == rhs.phone &&
-                lhs.outcome == rhs.outcome
+                lhs.outcome == rhs.outcome &&
+                lhs.reported == rhs.reported
     }
```

Comment:
I think we can remove these `==` and `hashValue` methods right?  In Swift 4.1 these should automatically use all members if the object is `Equatable`/`Hashable`.